jobserver/views/projects.py

```diff
@@ -111,7 +110,6 @@ def get_context_data(self, **kwargs):
         )
 
 
-@method_decorator(require_superuser, name="dispatch")
```

Comment:
Just double-checking: is it intentional that no permissions are required anywhere in the view?

Reply:
Yep, this was an oversight previously!